python-threatexchange/threatexchange/cli/match_cmd.py

```diff
@@ -51,6 +52,10 @@ class MatchCommand(command_base.Command):
     $ threatexchange match text -- This is my cool text
     ```
 
+    # Additional options: 
+    --rotation: For photo content, generate and match all 8 simple rotations 
+                (0°, 90°, 180°, 270°, flip X, flip Y, flip diagonal +1, flip diagonal -1)
+
```

Comment:
blocking: Remove this, we can document the arguments better with argparse features. The rest of this docbloc is meant to explain how the command works in general, not to document specific fields. check out `--help` to see how the options are dispalyed.